docker/Dockerfile

```diff
@@ -55,7 +55,7 @@ RUN cd /var/tmp && \
     rm -rf file-${FILE_VERSION}*
 
 # Install FITS
-COPY target/dependency/fits-*.zip /opt/fits.zip
+COPY target/fits-*.zip /opt/fits.zip
```

Comment:
If you're going to change the Dockerfile to work like this, then you should change the maven command in the README, so that the image can still be built normally.

Reply:
This is an interim step to accommodate the new script which, unfortunately for the time being, depends on the private LTS repository. Eventually FITS and FITSServlet (both WAR, ZIP and Docker images) will be going to public repositories. However, it was decided in scrum this morning, that those tasks will go into a future sprint.

Reply:
Done

Reply:
Does this work? I can't try it right now, but it looks different from the one in your script:

```shell
mvn dependency:get -DgroupId=edu.harvard.huit.lts -DartifactId=fits -Dversion=\"$FITS_VERSION\" \
-Dpackaging=zip \
-DremoteRepositories=https://artifactory.lts.harvard.edu/artifactory/lts-libs-all \
-Dtransitive=false \
-Ddest=target/
```

I would have expected it to have been the same, minus the `-DremoteRepositories` option

Reply:
Right, I was just asking if the update you made to the readme worked, because it wasn't what I had expected because it's different from the command you're running in your new script.

Reply:
In testing, I found: 
- `mvn dependency:get -DincludeArtifactIds=fits -DincludeTypes=zip` 
  - Does not work.
- `mvn dependency:copy-dependencies -DincludeArtifactIds=fits -DincludeTypes=zip` 
  - Does not work (copies to `target/dependency`, not `target/`).
-  `mvn dependency:get -DgroupId=edu.harvard.huit.lts -DartifactId=fits -Dversion=\"$FITS_VERSION\" \
-Dpackaging=zip \
-DremoteRepositories=https://artifactory.lts.harvard.edu/artifactory/lts-libs-all \
-Dtransitive=false \
-Ddest=target/`
   - Does not work (because it can't find `nz.govt.natlib:metadata:jar:3.6GA` and `com.portalmedia:embarc:jar:0.2`. More disappearing dependencies?)
   
For now, I went with a combination of:
`mvn dependency:copy-dependencies -DincludeArtifactIds=fits -DincludeTypes=zip`
`cp target/dependency/fits-1.6.0.zip target/.`

I also had a problem with `mvn -DskipTests clean package`, because it couldn't find `org.aes:aes-model:jar:1.5.0_06-64` and `edu.harvard.hcl:audiomarker:jar:1.6.0_65-b14-462-11M4609`. I manually deployed those to `lts-ext-release-local` to get it to work. 

If anyone has `nz.govt.natlib:metadata:jar:3.6GA` and `com.portalmedia:embarc:jar:0.2`, I can push those too.

Reply:
Honestly, I think this script is unsound. It could be used to build an image for any combination of FITS and the servlet, when it should be driven by the versions that are in the project. For example, if you want to build a specific version of the servlet, then you checkout the tag for that version and run the script. It then pulls the correct version of FITS, as defined in the pom.

This is what happens when you run:

```shell
mvn dependency:copy-dependencies -DincludeArtifactIds=fits -DincludeTypes=zip
```

Which is the command that I'd originally intended to be run. @cvicary there should be no need to do the subsequent `cp`. The Dockerfile should be reverted to its original form that expected the zip to be located in `target`.

Reply:
Also, I don't care what you all decide to do with the script. My only concern is that the Dockerfile remains usable to non-Harvard folks, and that instructions in the README are correct.